src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/PooledObjects/PooledArrayBuilder`1.cs

```diff
@@ -95,6 +95,11 @@ public void AddRange(IEnumerable<T> items)
         _builder.AddRange(items);
     }
 
+    public readonly Enumerator GetEnumerator()
+        => _builder is { } builder
+            ? new Enumerator(builder)
```

Comment:
So does `builder` need to be nullable in the `Enumerator`?

Reply:
The constructor doesn't need to be nullable. I'll tweak that.